docs/ref/contrib/gis/install/index.txt

```diff
@@ -159,7 +159,9 @@ new line in ``ld.so.conf``.  This is *one* example of how to do so::
 For OpenSolaris users, the system library path may be modified using the
 ``crle`` utility.  Run ``crle`` with no options to see the current configuration
 and use ``crle -l`` to set with the new library path.  Be *very* careful when
-modifying the system library path::
+modifying the system library path:
+
+.. code-block:: python
```

Comment:
Is there a way of skipping these changes? 

(Old-as-time, an rst codeblock  marked with `::` is Python.)

Reply:
I opened https://github.com/adamchainz/blacken-docs/issues/195 to see what @adamchainz says.

Reply:
@carltongibson I understand that it's @jvzammit who has taken the time to manually change all the `::` into `code-block` statements in order to get them linted :)

We also agreed that it's explicit rather than implicit.. and we like that according to our Zen :)

Reply:
Grrr. Right I missed that. 

> ...so it could be argued that blacken-docs should be able to know about the default and lint blocks that are unmarked?

That's more on the money to my mind. We're basically saying to change an ancient RST convention otherwise. I've adjusted the blacken-docs issue to see what Adam thinks.